test_trained_model.py

```diff
@@ -41,6 +41,9 @@ def test_trained_model(config_file: str = None, chosen_model: torch.nn.Module =
         else:
             raise ValueError("Unknown output type", output_type[item])
         config["NeuralNetwork"]["Architecture"]["output_dim"].append(dim_item)
+    config["NeuralNetwork"]["Architecture"]["output_type"] = config["NeuralNetwork"][
```

Comment:
Why do we need to add to the config?

Reply:
Because different NN structures are designed for graph and node outputs.